docker-compose.yml

```diff
@@ -17,24 +26,23 @@ services:
       interval: 5s
       timeout: 60s
       retries: 5
-  emx2:
+  molgenis-emx2:
     image: molgenis/molgenis-emx2
-    environment:
-    - MOLGENIS_POSTGRES_URI=jdbc:postgresql://postgres/molgenis
-    - MOLGENIS_POSTGRES_USER=molgenis
-    - MOLGENIS_POSTGRES_PASS=molgenis
-    #      - MOLGENIS_OIDC_CLIENT_ID=[]
-    #      - MOLGENIS_OIDC_CLIENT_SECRET=[]
-    #      - MOLGENIS_OIDC_CLIENT_NAME=MolgenisAuth
-    #      - MOLGENIS_OIDC_DISCOVERY_URI=https://auth.molgenis.org/.well-known/openid-configuration/
-    #      - MOLGENIS_OIDC_CALLBACK_URL=http://localhost:8080
+    container_name: molgenis-emx2
+    env_file: .env
     ports:
-    - "8080:8080"
+    - ${MOLGENIS_PORT}:${MOLGENIS_PORT}
     depends_on:
       postgres:
         condition: service_healthy
     restart: on-failure
-  ssr-catalogue:
+  molgenis-ssr-catalogue:
```

Comment:
what if you want a emx2 without a catalogue ? , separate compose files ? , comment out by default ?